<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGui</class>
 <widget class="QMainWindow" name="MainGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainGui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ImageView" name="imageview" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="snap">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Snap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="live">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Live</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DMD_Calibrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Perform calibration between camera image and DMD.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Calibrate DMD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_movie">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Movie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mouse_shoot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Click and light</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exposureT">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Binning</string>
    </property>
   </widget>
   <widget class="QComboBox" name="binning">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2x2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n images</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nAverage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detectCells">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Using CellPose to detect neuronal somas automatically. &lt;/p&gt;&lt;p&gt;CellPose Diameter set the minimal size of soma.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Detect cells</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MinMax size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minsize">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxsize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>70</string>
    </property>
   </widget>
   <widget class="QPushButton" name="somasStims">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run SLM stimulation to all detected somas in the DMD_dir&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Stimulate all cells</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ManualGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Manually grouping detected cells by mouse click&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Click and group</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CellPose Diameter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cellpose_diameter">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RandGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Random grouping</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="group_size">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Group size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createProtocol">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Protocol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadProtocol">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Protocol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runProtocol">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Protocol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadTest">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load test culture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopProtocol">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop protocol</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTab">
    <property name="title">
     <string>Tab</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTab2">
    <property name="title">
     <string>Tab2</string>
    </property>
   </widget>
   <addaction name="menuTab"/>
   <addaction name="menuTab2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
